auth/azcli.go

```diff
@@ -36,29 +86,15 @@ func (a *AzureCliAuthorizer) Token() (*oauth2.Token, error) {
 		return nil, fmt.Errorf("could not request token: conf is nil")
 	}
 
-	var token struct {
-		AccessToken string `json:"accessToken"`
-		ExpiresOn   string `json:"expiresOn"`
-		Tenant      string `json:"tenant"`
-		TokenType   string `json:"tokenType"`
-	}
-
-	var resourceType string
-	switch a.conf.Api {
-	case MsGraph:
-		resourceType = "ms-graph"
-	case AadGraph:
-		resourceType = "aad-graph"
-	}
-
-	azArgs := []string{"account", "get-access-token", fmt.Sprintf("--resource-type=%s", resourceType)}
+	azArgs := []string{"account", "get-access-token", fmt.Sprintf("--resource=%s", a.conf.Endpoint)}
```

Comment:
since this is a URI (and could be a Resource containing values which should be encoded) - should we quote this value for now?

```suggestion
	azArgs := []string{"account", "get-access-token", fmt.Sprintf("--resource=%q", a.conf.Endpoint)}
```

Reply:
Reading the docs and a couple of related issues, it looks like exec.Command() performs implicit quoting? i.e. manually quoting seems to produce extra escaped quotes in the arg.